data/changelog.txt

```diff
@@ -61,14 +62,21 @@ Adds 'What the hell happened' stories to NPCs, keyed off backgrounds.
 Feral Humans with blob psychosis.
 Allow plank and branch bundles to be made from cordage, vines, or wire.
 Microlabs can contain artifacts in containment.
-Add NPC_class and NPC json templates and instructions.
+Adds NPC_class and NPC json templates and instructions.
 Adapts almost all vehicle definitions to use directional parts (most of the work by Curstwist).
 Update UltiCa tileset and make it default.
-Add a number of flying zombies.
+Adds a number of flying zombies.
 Adds a new alternate needs panel to the label mode.
 Added many proficiencies and uses for them.
-Add frog mother monster.
+Adds frog mother monster.
 Adds mega mutant monsters.
+Adds a selection of mutated woodland creatures.
+New 'phase skulker' zombies teleport and haunt between the hallways of microlabs.
+Make mutant organs and lungs contain toxins.
+Adds Streetlights to sidewalks; traffic lights, stop lines and pedestrian crossings to intersections.
+Adds Rust Zombies to the Game.
+Diversify venom sources and effects.
```

Comment:
```suggestion
Adds streetlights to sidewalks, and traffic lights, stop lines, and pedestrian crossings to intersections.
Adds Rust Zombies.
Diversifies venom sources and effects.
```

Reply:
keep the semicolon otherwise the sentence structure's readability dips